core/themes/seven/css/style.css

```diff
@@ -780,9 +780,16 @@ div.teaser-checkbox .form-item,
 .text-format-wrapper .form-item {
   padding-bottom: 0;
 }
+.form-item {
```

Comment:
This is such a generic CSS selector, it can't be related to this issue only...

Reply:
IIRC these base form styles were missing from seven. It became obvious when all the form elements were added to the exposed filter on the modules page. Since we're only adding the one, it might be safe to remove this. I'll give it a try...

Reply:
yep. fixed.